.gitignore

```diff
@@ -105,8 +105,393 @@ venv.bak/
 /.vs
 /staged-data-analytics
 
-# VS Code
-.vscode
-
 # vi
 *~
+
+# julia binaries
+Research/julia/src/cloud-to-device-console-app/bin*
+Research/julia/src/cloud-to-device-console-app/obj/*
+Research/julia/src/ams-asset-player/bin*
+Research/julia/src/edge/modules/objectCounter/bin
```

Comment:
How many of the changes in this file are manual changes to omit your specific files vs things you copied from something? I see a lot of cruft here with commit messages etc....

Reply:
Beyond that, it looks like other projects have project-specific git ignores which you should probably model.

Reply:
Committing those changes now!